utils/test/plugins.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import { validateNamespacedId } from '../plugins';
+
+describe( 'validatePluginId', () => {
+	it( 'accepts a valid plugin id', () => {
+		expect( validateNamespacedId( 'gutenberg/plugin' ) ).toBe( true );
+	} );
+
+	it( 'rejects a pluginId with special characters besides a single "/"', () => {
+		const valid = validateNamespacedId( 'gutenberg//plugin' );
+		expect( console ).toHaveErrored();
+		expect( valid ).toBe( false );
+	} );
+
+	it( 'rejects a pluginId that\'s a number', () => {
```

Comment:
Nit: The intent of the validation is not to exclude numbers specifically, it's to exclude non-strings. The test description could be improved to reflect this.